docs/learn/learn-assets-guides.md

```diff
@@ -1,13 +1,15 @@
 ---
-id: learn-statemint
```

Comment:
Please implement redirects for the statemint doc as well, as it may have been referenced in third party websites